docs/content/config/security/ssl.md

```diff
@@ -385,7 +385,7 @@ The following example is the [basic setup][acme-companion::basic-setup] you need
         container_name: nginx-proxy-acme
         restart: always
         environment:
-          # Only docker-compose v2 supports: `volumes_from: [nginx-proxy]`,
+          # Only Docker Compose v2 supports: `volumes_from: [nginx-proxy]`,
```

Comment:
This was about the v2 schema, which is probably going to be easy to confuse with Compose V2 (CLI).

[`volumes_from`](https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes_from) (v2 schema) was [dropped in the v3 schema](https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes). However... `docker compose` (V2) may actually support a mix of v2 + v3 schema :sweat_smile: 

**EDIT:** Yup, [here we go](https://docs.docker.com/compose/compose-file/):

> The Compose spec **merges the legacy `2.x` and `3.x` versions**, aggregating properties across these formats and is implemented by Compose 1.27.0+.

- This is when the `version` field no longer became necessary IIRC.
- The docs old compose v2 + v3 references are under the left side-bar legacy item, while the reference we want for this information [now belongs here](https://docs.docker.com/compose/compose-file/05-services/#volumes_from), where we can see `volumes_from` is supported again.

---

Thus AFAIK, this note was about an alternative way with `volumes_from` that worked (_and now does again_), while the documented alternative is some ENV based feature `nginx-proxy-acme` must implement :sweat_smile: 

I think it's fine to drop this line?

```suggestion
```

Or it could provide a bit more context:

```suggestion
          # When `volumes_from: [nginx-proxy]` is not supported,
```

Reply:
Resolved in [2f9358d](https://github.com/docker-mailserver/docker-mailserver/pull/3295/commits/2f9358d587e6f5b6ce332e9fff760c560b60a4d6)